lib/ui/src/modules/ui/components/left_panel/stories/tree_node_type.js

```diff
@@ -0,0 +1,5 @@
+export default {
+  NAMESPACE: 'namespace',
+  COMPONENT: 'component',
+  STORY: 'story',
```

Comment:
maybe it's worth to add some comments about new UI structure. I'd like to dig a bit this (and previous) PR so it could help to review